danPortfolio.html

```diff
@@ -0,0 +1,88 @@
+			<article class="brief-intro">
+				<h3>Hello there!</h3>
+				<h1>Welcome to Dan's portfolio</h1>
+				<p>I write code</p>
+			</article>
+		</div>
+	</section>
+		
+	<section class="about" id="about">
+		<div class="box container-for-image-and-text-in-about">
+			<article class="container-for-text-in-about">
+				<h2>About</h2>
+					<p>Hi, I'm Dan, a full-stack programmer and graduate of i0 Academy, based currently just outside the beautiful city of Bath.</p>
+					<p>Feel free to browse the projects below to gauge my programming experience.</p>
+					<p>Outside coding, I enjoy playing the guitar, exercising, and reading.</p>  
+			</article>
+			<div class="container-for-image-in-about">
+				<img id="photo-of-me" img src="https://lh3.googleusercontent.com/a/AEdFTp6WY4Ej7I-rAKYjfmsA3DnIcXg3Ad3tTjwqu6sARQ=s576-p-no" alt="a photo of Dan" width="175px" height="#" style="#">
+			</div>
+		</div>
+	</section>
+
+	<section class="portfolio" id="portfolio">
+		<div class="box">
+			<h2>Portfolio</h2>
+			<p>Here are some projects in which I was involved.</p>
+			<div class="project-examples">
+				<h4>Project title</h4>
+				<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+				<article class="project-description">In this project, we etc.</article>
+			</div>
+			<div class="project-examples">
+					<h4>Project title</h4>
+					<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+					<article class="project-description">In this project, we etc.</article>
+				</div>
+				<div class="project-examples">
+					<h4>Project title</h4>
+					<img img src="https://cdn-developer-wp.arc.dev/wp-content/uploads/2022/06/coding-programming-project-ideas-1128x635.jpg" alt="image representing our project" width="100px" height="#" style="#">
+					<article class="project-description">In this project, we etc.</article>
+				</div>
+			</div>
+	</section>
+
+	<section class="contact" id="contact">
+		<h2>Contact</h2>
+		<p>The best way to get in touch with me is via email, but if you prefer social media I have included those too.</p>
+		<a href="#">
+			<img class="social-link" src="https://icon.signature.email/social/email-circle-small-1493b3-FFFFFF.png" alt="email-link" style="#">
+		</a>
+		<a href="#">
+			<img class="social-link" src="https://icon.signature.email/social/linkedin-circle-small-0077b5-FFFFFF.png" alt="linkedin-link" style="#">
+		</a>
+		<a href="#">
+			<img class="social-link" src="https://icon.signature.email/social/github-circle-small-24292e-FFFFFF.png" alt="github-link" style="#">
+		</a>
+	</section>
+
+	<footer>
+		<p>&copy 2023</p>
```

Comment:
we talked about maybe including 'contact' in the footer

Reply:
I reckon I'm just gonna have "back to top" and necessary copyright there